examples/integration_test.py

```diff
@@ -313,11 +314,20 @@ def verify_avro():
                   (msg.topic(), msg.partition(), msg.offset(),
                    msg.key(), msg.value(), tstype, timestamp))
 
+            # verify Falsy keys and values are consumed
```

Comment:
I think it would be better to verify all messages in a more generic way using the combinations list.
We know what we sent (the `combo`), so we know what to expect.

Reply:
that's fair, I was originally trying avoid post-processing keys and values with empty fields. The latest commit does exactly that and asserts that each combo matches each populated message field